index.html

```diff
@@ -15,19 +14,24 @@
                 <h1 class="inside-name">Kelvin</h1>
 
             </div>
-            <nav>
-                <ul class="nav-menu">
-                       <li><a href="#" class="item-home">Portfolio</a></li>
-                        <li><a href="#" class="item-about">About</a></li>
-                        <li><a href ="#"class="item-contact">Contact</a></li>
-                        <li><a href="#"class="item-email">
+            <nav class="nav-menu">
+                <button class="close">
+                    <i class="fa-solid fa-xmark"></i>
+                </button>
+                <ul>
+                       <li><a href="https://unleashedicon.github.io/personal-portfolio-project/" class="item-home">Portfolio</a></li>
+                        <li><a href="https://unleashedicon.github.io/personal-portfolio-project/" class="item-about">About</a></li>
+                        <li><a href ="https://unleashedicon.github.io/personal-portfolio-project/"class="item-contact">Contact</a></li>
+                        <li><a href="https://unleashedicon.github.io/personal-portfolio-project/"class="item-email">
                             <img src="images/Disabled.png"alt="Disabled">
                         </a></li>
                        
                     </ul>
```

Comment:
- [ ] Kindly remove the Disabled.png image icon since this is not a include on the original design 🚮 

- [ ] Nice implementing the design, however, The hamburger button and menu follow the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, text, margins). 
- Eg: Actually you used white text color "fff"  for your nav menu instead of `#FFF5E1;` and for thin bar after the nav menu you have used  "border-bottom: 1px solid #fff;"  instead of `#6F6C6B;` and so right now... Kindly make sure you stick to the original design as much as possible 🎨 🖌️